src/datastore/postgres-store.ts

```diff
@@ -2925,7 +2925,7 @@ export class PgDataStore
         SELECT ${TX_COLUMNS}, ${abiColumn()}
         FROM txs
         WHERE index_block_hash = $1 AND canonical = true AND microblock_canonical = true
-        ORDER BY microblock_sequence ASC, tx_index ASC
+        ORDER BY microblock_sequence DESC, tx_index DESC
```

Comment:
This function is only used by a Rosetta endpoint right? Are you sure this has the intended effect?

Reply:
Yeah, I pushed this because thought this would optimize the query but I didn't realize ASC is the necessary order here. I'll revert this change.